tests/foreman/cli/test_computeresource_libvirt.py

```diff
@@ -106,90 +106,70 @@ def libvirt_url():
-    compute_resource = module_target_sat.cli_factory.compute_resource(
+    assert cr['name'] == name
+    assert cr['provider'] == provider
+    assert cr['description'] == desc
+    assert cr['url'] == libvirt_url
+    assert cr['locations'][0] == module_location.name
+    assert cr['organizations'][0] == module_org.name
+    # UPDATE
+    new_name = gen_string('alphanumeric')
+    new_desc = gen_string('alphanumeric')
+    new_org = module_target_sat.cli.Org.create({'name': gen_string('alpha')})
+    new_loc = module_target_sat.cli.Location.create({'name': gen_string('alpha')})
+    cr_update = module_target_sat.cli.ComputeResource.update(
         {
-            'name': name,
-            'provider': FOREMAN_PROVIDERS['libvirt'],
-            'url': libvirt_url,
+            'id': cr['id'],
+            'new-name': new_name,
+            'description': new_desc,
+            'location-ids': new_loc['id'],
+            'organization-ids': new_org['id'],
         }
     )
-    # factory already runs info, just check the data
-    assert compute_resource['name'] == name
-
-
-@pytest.mark.tier1
-def test_positive_list(libvirt_url, module_target_sat):
-    """Test Compute Resource List
-
-    :id: 11123361-ffbc-4c59-a0df-a4af3408af7a
-
-    :expectedresults: Compute resource List is displayed
-
-    :CaseImportance: Critical
-    """
-    comp_res = module_target_sat.cli_factory.compute_resource(
-        {'provider': FOREMAN_PROVIDERS['libvirt'], 'url': libvirt_url}
-    )
-    assert comp_res['name']
-    result_list = module_target_sat.cli.ComputeResource.list(
-        {'search': 'name={}'.format(comp_res['name'])}
-    )
-    assert len(result_list) > 0
-    result = module_target_sat.cli.ComputeResource.exists(search=('name', comp_res['name']))
-    assert result
-
-
-@pytest.mark.tier1
-@pytest.mark.upgrade
-def test_positive_delete_by_name(libvirt_url, module_target_sat):
-    """Test Compute Resource delete
-
-    :id: 7fcc0b66-f1c1-4194-8a4b-7f04b1dd439a
-
-    :expectedresults: Compute resource deleted
-
-    :CaseImportance: Critical
-    """
-    comp_res = module_target_sat.cli_factory.compute_resource(
-        {'provider': FOREMAN_PROVIDERS['libvirt'], 'url': libvirt_url}
-    )
-    assert comp_res['name']
-    module_target_sat.cli.ComputeResource.delete({'name': comp_res['name']})
-    result = module_target_sat.cli.ComputeResource.exists(search=('name', comp_res['name']))
-    assert len(result) == 0
+    # READ
+    cr_read = module_target_sat.cli.ComputeResource.info({'id': cr_update[0]['id']})
+    assert cr_read['name'] == new_name
+    assert cr_read['description'] == new_desc
+    assert cr_read['provider'] == provider
+    assert cr_read['url'] == libvirt_url
+    assert cr_read['locations'][0] == new_loc['name']
+    assert cr_read['organizations'][0] == new_org['name']
+    # LIST
+    cr_list = module_target_sat.cli.ComputeResource.list({'search': f'name={cr_read["name"]}'})
+    assert len(cr_list) > 0
```

Comment:
List should return exact one CR object, as we are now creating only one CR and also searching with `new_name`, right?
```suggestion
    assert len(cr_list) == 1
```